addons/mail/models/mail_thread.py

```diff
@@ -4528,7 +4528,7 @@ def _message_update_content(self, message, body, attachment_ids=None, partner_id
         empty_messages._cleanup_side_records()
         empty_messages.write({'pinned_at': None})
         res = {
-            "attachments": Store.many(message.attachment_ids.sorted("id")),
+            "attachment_ids": Store.many(message.attachment_ids.sorted("id")),
```

Comment:
So updating message with attachment is currently broken as non-internal user? Or the whole method is called as sudo?
I guess I'll need to check that.